editor-components/modal/confirmation-modal-1/confirmation.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from "react";
+import ComposerModalClose from "../../../composer-base-components/close/close";
+import { BaseModal, TypeUsableComponentProps } from "../../EditorComponent";
+import styles from "./confirmation.module.scss";
+import ComposerLink from "../../../../custom-hooks/composer-base-components/Link/link";
+import { ComposerIcon } from "../../../composer-base-components/icon/icon";
+
+
+
+
+
+class ConfirmationModal extends BaseModal{
+    constructor(props?: any) {
+        super(props, styles);
+
+        
+
+        this.addProp({
+            type: "image",
+            key: "image",
+            displayer: "image",
+            value: "https://picsum.photos/200"
```

Comment:
you cannot access a random picsum photo for every time. select one photo and get its spica storage url. then insert it here